tools/__init__.py

```diff
@@ -237,17 +244,14 @@ def executable_path(self):
 
     @property
     def _package_installed(self):
-        result = util.misc.run_and_print(["conda", "list", "-f", "-c", "-p", self.env_path, "--json", self.package], silent=True, env=self.conda_env, buffered=True)
+        result = util.misc.run_and_print(["conda", "list", "-f", "-c", "-p", self.env_path, "--json", self.package], silent=True, env=self.conda_env)
         if result.returncode == 0:
             command_output = result.stdout.decode("UTF-8")
             data = json.loads(self._string_from_start_of_json(command_output))
             if len(data) > 0:
                 return True
         return False
 
-    def is_installed(self):
-        return self.installed
```

Comment:
CondaPackage had this version of is_installed previous to this PR, which returns self.installed, which is set by the verify_install() which manually memoizes its return value.